src/days-2-4-adv/adv.py

```diff
@@ -49,3 +49,41 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
+class Room():
+    def __init__(myRoom,name,description):
+        myRoom.name = name
+        myRoom.description = description
+        myRoom.currentRoom= 0
+        myRoom.priorRoom = 0
+
```

Comment:
I am guessing that this is something in progress because I see this is also in a separate room.py file.